projects/canopy/src/lib/forms/radio/radio-button.component.scss

```diff
@@ -1,5 +1,12 @@
 @import '../../../styles/mixins';
 
+.lg-radio-button {
```

Comment:
this can be all in a line without being split

Reply:
It can, but my first thought was that we could leverage nesting so we could have a top-level `.lg-radio-button` with the suffixes (`&__content, &--radio, &__input`, etc) used for more targeted selection. If it seems more clear though, having it on one line, I will change it, let me know 😄.

Reply:
You could do that but in this case you have only added the `&__content` so either you add everything else or leave the selector as `lg-radio-button__hint`.

Reply:
Reverted it.